fitbit/api.py

```diff
@@ -477,8 +473,7 @@ def water_goal(self, target=None):
         """
         Implements the following APIs
 
-        https://wiki.fitbit.com/display/API/API-Get-Water-Goal
-        https://wiki.fitbit.com/display/API/API-Update-Water-Goal
+        https://dev.fitbit.com/docs/food-logging/#get-water-goal
```

Comment:
Please add the update URL
https://dev.fitbit.com/docs/food-logging/#update-water-goal